libs/ledger-live-common/src/families/cosmos/banner.ts

```diff
@@ -0,0 +1,79 @@
+import { getCurrentCosmosPreloadData } from "./preloadedData";
+import { getAccountBridge } from "../../bridge";
+import { LEDGER_VALIDATOR_ADDRESS } from "./utils";
+import type { CosmosAccount } from "./types";
+
+export async function getAccountBannerState(account: CosmosAccount) {
```

Comment:
Can you please add the return type for this function?

Reply:
Done